--- /www/config/control_panel.cgi.orig
+++ /www/config/control_panel.cgi
@@ -209,8 +209,8 @@
 
 if { "[read_var /etc/config/tweaks CP_DEVCONFIG]" != "" } {
   puts "<td><div class=\"cpButton\">"
-  puts "<div class=\"StdTableBtn CLASS21701\" onclick=\"window.open('/tools/devconfig.cgi?sid=$sid');\">devconfig</div>"
-  puts "<div class=\"StdTableBtnHelp\"></div></td>"
+  puts "<div class=\"StdTableBtn CLASS21701\" onclick=\"window.open('/tools/devconfig.cgi?sid=$sid');\">DevConfig</div>"
+  puts "<div class=\"StdTableBtnHelp\"><img id=\"showCouplingCPHelp\" src=\"/ise/img/help.png\"></div></td>"
   incr i
 
   if { $i == $COL_COUNT } {
