--- /www/config/cp_maintenance.cgi.orig
+++ /www/config/cp_maintenance.cgi
@@ -350,7 +350,8 @@
     set styleMaxWidth ""
     #set styleMaxWidth "style=max-width:70px;"
 
-    table {class="popupTable CLASS20901 j_translate"} {border="1"} {
+    division {style="height:80vh;width:100%;overflow:auto;"} {
+    table {class="popupTable CLASS20901 j_translate"} {border="1"} {height="100%"} {
       table_row {class="CLASS20902"} {
         table_data {class="CLASS20903"} $styleMaxWidth {
           #puts "Zentralen-<br>"
@@ -624,6 +625,7 @@
         }
       }
     }
+    }
     checkIfFwOnly
   }
   division {class="popupControls"} {
--- /www/config/cp_network.cgi.orig
+++ /www/config/cp_network.cgi
@@ -208,7 +208,8 @@
     puts "\${dialogSettingsNetworkTitle}"
   }
   division {class="CLASS21114 j_translate"} {
-    table {class="popupTable"} {border=1} {width="100%"} {
+    division {style="height:80vh;width:100%;overflow:auto;"} {
+    table {class="popupTable"} {border=1} {width="100%"} {height="100%"} {
       table_row {class="CLASS21115"} {
         table_data {class="CLASS21116"} {
           #puts "IP-<br/>Einstellungen"
@@ -412,6 +413,7 @@
         }
       }
     }
+    }
   }
   division {class="popupControls"} {
     table {
--- /www/config/cp_security.cgi.orig
+++ /www/config/cp_security.cgi
@@ -935,7 +935,8 @@
   }
 
   division {class="CLASS20815"} {
-    table {class="popupTable j_translate"} {border="1"} {
+    division {style="height:80vh;width:100%;overflow:auto;"} {
+    table {class="popupTable j_translate"} {border="1"} {height="100%"} {
       table_row {class="CLASS20806"} {
         table_data {class="CLASS20807"} {
           puts "\${dialogSettingsSecurityTDKey}"
@@ -1302,6 +1303,7 @@
         }
       }
     }
+    }
   }
   division {class="popupControls"} {
     table {
--- /www/config/cp_software.cgi.orig
+++ /www/config/cp_software.cgi
@@ -267,7 +267,8 @@
     puts "\${dialogSettingsExtraSoftwareTitle}"
   }
   division {class="CLASS21406"} {
-    table {class="popupTable"} {border="1"} {
+    division {style="height:80vh;width:100%;overflow:auto;"} {
+    table {class="popupTable"} {border="1"} {height="100%"} {
       set scripts ""
       set loop -1
       catch { set scripts [glob /etc/config/rc.d/*] }
@@ -408,6 +409,7 @@
         }
       }
     }
+    }
   }
   division {class="popupControls"} {
     table {
